### ISSUE TITLE
Le séparateur de date pour une période n'est pas conforme à ISO 8601

### ISSUE THREAD

**Issue by ColinMaudry** [etalab/format-subventions#2]
Le séparateur devrait être / et non _.

Source :
https://xkcd.com/1179/

Véritable source :
https://www.lexjansen.com/nesug/nesug06/cc/cc17.pdf (page 2)
https://fr.wikipedia.org/wiki/ISO_8601#Les_durées_et_intervalles

**Comment by CharlesNepote** [comment]
Merci Colin, je pense c'est important de prendre en compte ce point. Autant aller au bout et ISO 8601 est vraiment très répandu (mais je ne t'apprends sans doute rien).

Serait-il possible de le mettre en oeuvre rapidement, notamment pour améliorer l'alignement avec la spec du SCDL OpenDataFrance ?

**Comment by ColinMaudry** [comment]
Le soucis c'est que certains éditeurs de logiciels de gestion ont déjà fait des développements en utilisant la spec actuelle. Je vais les sonder pour voir si cette évolution leur parait faisable.

**Comment by CharlesNepote** [comment]
Bonjour Colin. Avez-vous avancé sur ce point ?

**Comment by ColinMaudry** [comment]
Non, notre attention chez Etalab est plutôt orientée vers une refonte du schéma, toujours en CSV. Mais il n'est pas évident de mobiliser les parties prenantes, déjà très sollicitées.

**Comment by ColinMaudry** [comment]
Du coup, pour l'instant, le sujet est en pause.
